.circleci/config.yml

```diff
@@ -100,6 +100,20 @@ jobs:
           name: Lint sources
           command: make lint
 
+  plugins:
+    docker:
+      - image: circleci/node:14
+    working_directory: ~/fun/src/plugins
+    steps:
+      - checkout:
+          path: ~/fun
+      - run:
+          name: Install dependencies
+          command: yarn install --frozen-lockfile
+      - run:
+          name: Build plugins, lint sources and run tests
+          command: yarn build
+
```

Comment:
Could you a comment to describe the utility of this job?

Reply:
Good idea) Thanks) added a comment)